doc/nrf/known_issues.rst

```diff
@@ -552,6 +552,16 @@ Front-End Modules API implementation missing
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+DRGN-11963: LL control procedures cannot be initiated at the same time
+The following list of LL control procedures cannot be initiated at the same time:
+ * LE start encryption
+ * LE read remote features
+ * LE read remote version
+ * LE connection parameter update
```

Comment:
The list should be:
```suggestion
 * LE Connection Update procure
 * LE Channel Map Update procedure
 * LE Encryption procedure
```

Reply:
Updated list, channel map did not belong there either.